.github/workflows/pr-autofix.yml

```diff
@@ -42,6 +42,6 @@ jobs:
         run: yarn build
         if: steps.paths_src_.outputs.changed == 'true'
 
-      - uses: Khan/autofix-commit-action@master
+      - uses: Khan/autofix-commit-action@main
```

Comment:
**note:** Caught this because I was still getting Node16 deprecation warnings after fixing that in the `Khan/autofix-commit-action` repo.